src/assets/scss/loaders.scss

```diff
@@ -0,0 +1,36 @@
+  .update .loader {
+      margin-top:30%;
+      margin-left: 40%;
+  }
+
+  .update .loader span{
+    display: inline-block;
+    width: 20px;
+    height: 20px;
```

Comment:
Why not nest this span within the above call since we are using sass?

Reply:
Use `rem` instead of `px`